opslevel/provider.go

```diff
@@ -99,7 +99,7 @@ func Provider() terraform.ResourceProvider {
 			opts = append(opts, opslevel.SetUserAgentExtra(fmt.Sprintf("terraform-provider-%s", version)))
 
 			if timeout > 0 {
-				opts = append(opts, opslevel.SetTimeout(time.Duration(timeout)))
+				opts = append(opts, opslevel.SetTimeout(time.Second*time.Duration(timeout)))
```

Comment:
I'm having issues with goland so can't introspect this package but what does time.Second do and why does multiplying with the Duration give you seconds? I'm assuming we want the user to be able to provide 10 which in time.Duration means 10ms and multiplying it with time.Second will convert it to 10s?

Reply:
nvm, just fixed goland and I was able to introspect and see that it's a const that's being used to convert milliseconds to seconds.

Reply:
Wait but now I'm seeing that duration is in nanoseconds?